src/code/audio_synthesis.c

```diff
@@ -4,6 +4,7 @@
 #define DEFAULT_LEN_1CH 0x1A0
 #define DEFAULT_LEN_2CH 0x340
 
+// Data Memory Addresses for the RSP
```

Comment:
```suggestion
// DMEM Addresses for the RSP
```
"dmem" is more suggestive than "data memory" imo

Reply:
Isn't `DMEM` an abbreviation for `data memory` (as opposed to `IMEM`, or instruction memory). So:
// DMEM (Data Memory) Addresses for the RSP?

Reply:
Sure, but it's a very common abbreviation, what I mean is "dmem" makes me think more of "the data memory on the rsp" than "data memory" does

To me it's like if instead of writing "RAM" you wrote "Random Access Memory" or "RAM (Random Access Memory)" :sweat_smile:

Reply:
Maybe I'm just biased because I decompiled the entirety of audio_synthesis.c from MM and it took me a while to realize what `DMEM` means